### PR TITLE
Added commonIdType

### PR DESCRIPTION
**Pull request by pstenbjorn**
@jungshadow I added commonIdType element and implementation.  Also following proposed guidelines in managing feature changes.

### CODE REVIEW

File: vip_spec.xsd [hunk 1]
```diff
@@ -407,6 +414,17 @@
       <xs:enumeration value="NO"/>
     </xs:restriction>
   </xs:simpleType> 
+  
+  <xs:simpleType name="commonIdEnum">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="ocdid"/>
+      <xs:enumeration value="fips"/>
+      <xs:enumeration value="OCDID"/>
```
Comment (jungshadow) [inline vip_spec.xsd]: Do we need multiple cases for ocdid (e.g. caps and lower)?

Comment (pstenbjorn) [inline vip_spec.xsd]: I would think not. And probably best if we don't now that I think of it.

File: vip_spec.xsd [hunk 2]
```diff
@@ -174,7 +174,8 @@
               <xs:element name="name" type="xs:string" />
               <xs:element minOccurs="0" name="type" type="xs:string" />
               <xs:element minOccurs="0" name="number" type="xs:integer" />
-			  <xs:element minOccurs="0"	name="common_id" type="xs:string" />
+              <xs:element minOccurs="0"	name="common_id" type="commonIdType" />
```
Comment (jungshadow) [inline vip_spec.xsd]: Not that this is a big deal, but is there a tab in between these attributes?

Comment (pstenbjorn) [inline vip_spec.xsd]: There is.  I replaced it in my local instance and committed it to this branch

**Comment by jungshadow** [comment]
Perfect.

**Comment by jungshadow** [comment]
@pstenbjorn Hmm...I think I merged too soon. I failed to notice that you hadn't removed the caps enumerations, yet. Want me to revert?

**Comment by pstenbjorn** [comment]
Yes please.

Let me know when it is reverted and I will recommit.

**Pull request merged**